### PR TITLE
Update getting started information

### LINKED ISSUE #9: Getting Started Page

### LINKED ISSUE #17: Getting Started Page
Feature #14 

## Description

The `Getting Started` page currently contains outdated information, based on the initial implementation where the communication with the EDC was done through an extension.

We want to update this page and mention that it provides a development UI for the `DataManagementAPI`.

## Acceptance Criteria

- [x] `Getting Started` page is updated to reflect the current state of implementation
- [x] Text contains information that it is a development UI for the `DataManagementAPI`
- [x] Page contains a link to the `DataManagementAPI` specification

## Tasks

- [x] Update text on the `Getting Started` page

### PR DESCRIPTION
**Pull request by marcgs**
closes agera-edc/DataDashboardFork#9 

Update getting started page in accordance to the current state of development

### CODE REVIEW

File: src/modules/edc-demo/components/introduction/introduction.component.html [hunk 1]
```diff
@@ -32,44 +30,33 @@ <h1>The Eclipse Dataspace Connector</h1>
...
-          [routerLink]="['/contractdefinition-editor']">Contract
-          Definition Editor</a></li>
-        <li>View which assets have been transfered in your Dataspace using the <a
-          [routerLink]="['/transfer-history-viewer']">Transfer History Viewer</a></li>
-        <li>Publish a new asset into your Dataspace using the <a [routerLink]="['/asset-publisher']">Asset Publisher</a>
-        </li>
+        <li>View the asset catalog available to you in your Dataspace using the <a [routerLink]="['/catalog-browser']" class="md-accent">Catalog Browser</a></li>
+        <li>Negotiate a contract for data sharing in your Dataspace using the <a [routerLink]="['/catalog-browser']">Catalog Browser</a></li>
+        <li>View your existing contracts using the <a [routerLink]="['/contract-viewer']">Contract Viewer</a></li>
+        <li>Transfer an asset in your Dataspace using the <a [routerLink]="['/contract-viewer']">Contract Viewer</a></li>
```
[the remark this answers is not in the document]
Comment (zeier) [reply src/modules/edc-demo/components/introduction/introduction.component.html:55]: `Viewer` has been removed from the other pages as well, as discussed previously (assumption is that the description and links will change once the corresponding pages are changed).

**Review by zeier — APPROVED** [review]

File: src/modules/edc-demo/components/introduction/introduction.component.html [hunk 2]
```diff
@@ -32,44 +30,33 @@ <h1>The Eclipse Dataspace Connector</h1>
...
-          are in the flux
-          on the connector side. The intention is to show the general flow (where possible and useful).
+          This EDC Data Dashboard is a developer UI for the EDC Data Management API.
+          This application is not intended for production usage and can be used to showcase EDC from a technical perspective, as the UI
+          is designed as a 1-1 mapping of the Data Management API.
         </p>
       <p>
         <a class="link" color="secondary"
-           href="https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/samples/02-health-endpoint/README.md"
+           href="https://github.com/agera-edc/DataSpaceConnector/tree/main/extensions/api/data-management"
```
[the remark this answers is not in the document]
Comment (chrislomonico) [reply src/modules/edc-demo/components/introduction/introduction.component.html:42]: Agreed; we should avoid referencing our fork here

**Review by chrislomonico — CHANGES_REQUESTED** [review]
please review open comments, will approve after

File: src/modules/edc-demo/components/introduction/introduction.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcgs) [reply src/modules/edc-demo/components/introduction/introduction.component.html]: Updated paths and instructions accordingly

**Review by chrislomonico — APPROVED** [review]

**Pull request merged**